addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -206,17 +221,30 @@ def migration_invoice_moves(env):
         FROM account_invoice_line ail
             JOIN account_invoice ai ON ail.invoice_id = ai.id AND ai.state NOT IN ('draft', 'cancel')
             JOIN account_move am ON ail.invoice_id = am.old_invoice_id
-        WHERE am.id = aml.move_id AND ail.company_id = aml.company_id AND ail.account_id = aml.account_id
-            AND ail.partner_id = aml.partner_id
-            AND ail.name = aml.name
```

Comment:
may be you could change it to three criterias instead of two, and putting the name in the more strictly one?